test/e2e/operator_test.go

```diff
@@ -2757,9 +2828,9 @@ func newNodePortController(name types.NamespacedName, domain string) *operatorv1
 	}
 }
 
-func newHostNetworkController(name types.NamespacedName, domain string) *operatorv1.IngressController {
+func newHostNetworkController(name types.NamespacedName, domain string, strategy *operatorv1.HostNetworkStrategy, placement *operatorv1.NodePlacement) *operatorv1.IngressController {
 	repl := int32(1)
-	return &operatorv1.IngressController{
+	ig := &operatorv1.IngressController{
```

Comment:
Most variables with type `operatorv1.IngressController` use the name `ic` or `ing`.  Is there a reason to use a different abbreviation here?